bundle.Dockerfile

```diff
@@ -1,20 +1,21 @@
 FROM scratch
-LABEL com.redhat.openshift.versions="v4.5-v4.7"
-LABEL com.redhat.delivery.backport=true
-LABEL com.redhat.delivery.operator.bundle=true
```

Comment:
these seem like they could be important?

Reply:
Also our DBaaS UX only works on 4.8+ so we should make sure that's included

Reply:
OK. I added them back and include 4.8